<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transmitter End</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Receiver End</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Transmit Image</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
